README.md

```diff
@@ -1,302 +1,32 @@
-
-I witness policy allowers administrators trace the compliance status of an artifact at any point during it's lifecycle.
-
-## Witness Verification
-
-### Verification Lifecycle
-
-![](docs/assets/verification.png)
-
-## Using [SPIRE](https://github.com/spiffe/spire) for Keyless Signing
-
-Witness can consume ephemeral keys from a [SPIRE](https://github.com/spiffe/spire) node agent. Configure witness with the flag `--spiffe-socket` to enable keyless signing.
-
-During the verification process witness will use the [Rekor](https://github.com/sigstore/rekor) integrated time to make a determination on certificate validity. The SPIRE certificate only needs to remain valid long enough for the attestation to be integrated into the Rekor log.
-
-## Witness Examples
-
-- [Using Witness To Prevent SolarWinds Type Attacks](examples/solarwinds/README.md)
-- [Using Witness To Find Artifacts With Hidden Vulnerable Log4j Dependencies](examples/log4shell/README.md)
-
-## Media
-
-- [Blog - What is a supply chain attestation, and why do I need it?](https://www.testifysec.com/blog/what-is-a-supply-chain-attestation/)
-- [Talk - Securing the Software Supply Chain with the in-toto & SPIRE projects](https://www.youtube.com/watch?v=4lFbdkB62QI)
-- [Talk - Securing the Software Supply Chain with SBOM and Attestation](https://www.youtube.com/watch?v=wX6aTZfpJv0)
-
-## Roadmap
-
-- Attestors for all major platforms
-- CaC Card Attestor
-- GovCloud Attestor
-- OIDC Attestor
-- FIDO Attestor
-- Vault Key Provider
-- Cloud KMS Support
-- Kubernetes Admission Controller
-- SIEM Collection Agent
-- Cosign Signature Validation
-- Notary v2 Signature Validation
-- [Zarf](https://github.com/defenseunicorns/zarf) Integration
-- IronBank Attestor
-
-## Support
-
-[TestifySec](https://testifysec.com) Provides support for witness and other CI security tools.
-[Contact Us](mailto:info@testifysec.com)
+## Status
+This library is currently pre-1.0 and therefore should be considered unstable and subject to change.
+
+## Features
+- Creation and signing of in-toto attestations
+- Verification of in-toto attestations and associated signatures with:
+  - Witness policy engine
+  - [OPA Rego policy language](https://www.openpolicyagent.org/docs/latest/policy-language/)
+- A growing list of attestor types defined under a common interface
+- A selection of attestation sources to search for attestation collections
+
+## Documentation
+For more detail regarding the library itself, we recommend viewing [pkg.go.dev](https://pkg.go.dev/github.com/testifysec/go-witness). For
+the documentation of the witness project, please view [the main witness repository](https://github.com/in-toto/witness/tree/main/docs).
+
+## Requirements
+In order to effectively contribute to this library, you will need:
+- A Unix-compatible Operating System
+- GNU Make
+- Go 1.19
```

Comment:
Side note: we should chat about minimum support Go version at some point. I think we use Go 1.21.x in most places.

Reply:
yeah for sure - I was actually looking to tackle that when getting the development environment configuration figured out and defined.